rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorerCleaner.java

```diff
@@ -30,11 +30,13 @@
     private PeerExplorer peerExplorer;
     private ScheduledExecutorService updateTask;
     private long updatePeriod;
+    private long cleanPeriod;
     private boolean running = false;
 
-    public PeerExplorerCleaner(PeerExplorer peerExplorer, long updatePeriod) {
+    public PeerExplorerCleaner(PeerExplorer peerExplorer, long updatePeriod, long cleanPeriod) {
         this.peerExplorer = peerExplorer;
         this.updatePeriod = updatePeriod;
+        this.cleanPeriod = cleanPeriod;
 
         this.updateTask = Executors.newSingleThreadScheduledExecutor(r -> new Thread(r, "PeerExplorerCleaner"));
```

Comment:
I suggest adding a comment here that single thread is important because the rest of the code assumes a single thread.

Reply:
:+1: